shop_project/shop_app/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+import datetime,os
+from django.contrib.auth.models import User
+# Create your models here.
+
+def getFileName(request,filename):
+    now_time=datetime.datetime.now().strftime("%Y%m%d%H:%M:%S:")
+    new_filename="%s%s"%(now_time,filename)
+    return os.path.join('uploads/',new_filename)
+
+class Category_Hari(models.Model):
```

Comment:
Follow proper naming conventions
It should be either be CategoryHari or category_hari
And why someone's name in code? Use relevant names to the application